tests/output_template/test_run.py

```diff
@@ -0,0 +1,13 @@
+import time
+from unittest import TestCase
+
+from run import _timerfunc
+
+
+class TestRun(TestCase):
+
+    def test_timerfunc(self):
+        def sleep(sec):
+            time.sleep(sec)
+        _, got = _timerfunc(sleep, [1])
+        self.assertEqual(int(got), 1)
```

Comment:
It's safer to use assertAlmostEqual.
```suggestion
        self.assertAlmostEqual(got, 1)
```

Reply:
@tfujiwar Thanks!!
In my understanding, `sleep(1)` will sleep at least 1 sec, and maybe longer than 1 sec, so `int(got)` is enough safe in (1<=got<2).

https://docs.python.org/3/library/time.html#time.sleep
> Also, the suspension time may be longer than requested by an arbitrary amount because of the scheduling of other activity in the system.
> Changed in version 3.5: The function now sleeps at least secs even if the sleep is interrupted by a signal, except if the signal handler raises an exception

In my environment, sleep took 1.001488673966378 seconds to complete. So if we use `assertAlmostEqual`, we need to set `places=<2`, but I am not sure about the good value of `places` in other situations. I am afraid that this test is fragile with a strict assertion.
Ummm, how about using `self.assertAlmostEqual(got, 1, places=1)`?

Reply:
Ah, you are right. `assertAlmostEqual` was too strict. Thank you.